docs/html/conf.py

```diff
@@ -28,27 +28,10 @@
     "sphinxcontrib.towncrier",
 ]
 
-# General information about the project.
-project = "pip"
-copyright = "The pip developers"
-
-# Find the version and release information.
-# We have a single source of truth for our version number: pip's __init__.py file.
-# This next bit of code reads from it.
-file_with_version = os.path.join(docs_dir, "..", "src", "pip", "__init__.py")
-with open(file_with_version) as f:
-    for line in f:
-        m = re.match(r'__version__ = "(.*)"', line)
-        if m:
-            __version__ = m.group(1)
-            # The short X.Y version.
-            version = ".".join(__version__.split(".")[:2])
-            # The full version, including alpha/beta/rc tags.
-            release = __version__
-            break
-    else:  # AKA no-break
-        version = release = "dev"
-
+copyright = copyright
+project = project
+version = version
+release = release
```

Comment:
Alternatively: add these to `__all__` in `docs/common_conf.py` and `import *` from there, but I preferred being explicit